doc/source/whatsnew/v1.5.0.rst

```diff
@@ -178,10 +178,87 @@ Styler
 
 - Fix showing "None" as ylabel in :meth:`Series.plot` when not setting ylabel (:issue:`46129`)
 
-.. _whatsnew_150.notable_bug_fixes.notable_bug_fix2:
+.. _whatsnew_150.notable_bug_fixes.rolling_moments:
 
-notable_bug_fix2
-^^^^^^^^^^^^^^^^
+rolling mean/sum/std/var/skew/kurt now output correct result for window with same values
```

Comment:
This can just be a 1 line description in the `Groupby/resample/rolling` section

Reply:
Did you mean to write like this?

![image](https://user-images.githubusercontent.com/48577571/162660341-fac1c893-c840-48dd-976b-84cdd7f550ee.png)

Reply:
Just a 1 line statement like the rest of the notes in this section 

``` - Bug in ... (:issue:`42064`) ```

Reply:
You can use the old line you had before creating this section.